File: app/src/test/java/com/bird/launcher/session/SessionContractTest.java
package com.bird.launcher.session;

import static org.junit.Assert.assertEquals;

import org.junit.Test;

public class SessionContractTest {
    @Test
    public void sessionConstantsMatchProviderContract() {
        assertEquals("com.bird.launcher.provider", SessionContract.AUTHORITY);
        assertEquals("session", SessionContract.Session.PATH);
        assertEquals("content://com.bird.launcher.provider/session", SessionContract.Session.CONTENT_URI_STRING);
        assertEquals("username", SessionContract.Session.COLUMN_USERNAME);
        assertEquals("logged_in", SessionContract.Session.COLUMN_LOGGED_IN);
    }
}

File: app/src/test/java/com/bird/StarrySkyTeaHouse/session/SessionContractTest.kt
package com.bird.StarrySkyTeaHouse.session

import org.junit.Assert.assertEquals
import org.junit.Test

class SessionContractTest {
    @Test
    fun sessionConstantsMatchProviderContract() {
        assertEquals("com.bird.StarrySkyTeaHouse.provider", SessionContract.AUTHORITY)
        assertEquals("com.bird.StarrySkyTeaHouse.permission.READ_SESSION", SessionContract.READ_PERMISSION)
        assertEquals("session", SessionContract.Session.PATH)
        assertEquals("content://com.bird.StarrySkyTeaHouse.provider/session", SessionContract.Session.CONTENT_URI_STRING)
        assertEquals("username", SessionContract.Session.COLUMN_USERNAME)
        assertEquals("logged_in", SessionContract.Session.COLUMN_LOGGED_IN)
    }
}
